docs/documenting-schema.rst

```diff
@@ -0,0 +1,73 @@
+.. _documenting-schema:
+
+Documenting schema
+=======================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
+
+Keeping your documentation close to the code that supports it is a great way to ensure that it is always accurate and up-to-date. To support this workflow, GraphQL allows programmers to write descriptions directly alongside types in `GraphQL Schema Definition Language syntax <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_.
```

Comment:
```suggestion
Keeping your documentation right next to its code is a great way to ensure it will always stay accurate and up-to-date. To support this workflow, GraphQL allows programmers to write descriptions directly alongside types in the `GraphQL Schema Definition Language syntax <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_.
```

Reply:
I appreciate the suggestions, @NyanKiyoshi.

I would like to defend the existing paragraph as grammatically correct, though.  It's not clear to me what your suggestions are trying to accomplish, so I'd just as soon keep the current version in this case.